runtime/model-protobuf/src/test/java/io/aklivity/zilla/runtime/model/protobuf/internal/ProtobufModelTest.java

```diff
@@ -481,4 +521,200 @@ public void shouldExtract()
         converter.extracted(int64Path, visitor);
 
     }
+
+    @Test
+    public void shouldWriteValidProto2MessageWithRequiredFields()
+    {
+        TestCatalogConfig catalog = CatalogConfig.builder(TestCatalogConfig::new)
+                .namespace("test")
+                .name("proto2test")
+                .type("test")
+                .options(TestCatalogOptionsConfig::builder)
+                .id(10)
+                .schema(PROTO2_SCHEMA)
+                .build()
+                .build();
+        when(context.supplyCatalog(catalog.id)).thenReturn(new TestCatalogHandler(catalog.options));
+
+        ProtobufModelConfig model = ProtobufModelConfig.builder()
+                .catalog()
+                .name("proto2test")
+                .schema()
+                .strategy("topic")
+                .version("latest")
+                .subject("proto2-value")
+                .record("Proto2Message")
+                .build()
+                .build()
+                .build();
+        ProtobufWriteConverterHandler converter = new ProtobufWriteConverterHandler(model, context);
```

Comment:
Bring back the nested indentation to convey structure of the model object being created by the builder.

Feedback applies to all similar usages of builder API.